share/commands/agent-check

```diff
@@ -80,7 +80,7 @@ check_and_fix_cfengine_processes() {
   fi
 
   # List the CFEngine processes running
-  CF_PROCESS_RUNNING=`${PS_COMMAND} | sed -ne "\\%${CFE_BIN_DIR}/\\(cf-execd\\|cf-agent\\)%p" | cat`
+  CF_PROCESS_RUNNING=`${PS_COMMAND} | grep -E "${CFE_BIN_DIR}/cf-(agent|execd)" | cat`
```

Comment:
we used to have egrep "${CFE_BIN_DIR}/(cf-execd|cf-agent)" 
see https://github.com/Normation/rudder-packages/pull/796/files